test/e2e/storage/persistent_volumes-local.go

```diff
@@ -150,15 +151,8 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 	)
 
 	ginkgo.BeforeEach(func() {
-		// Get all the schedulable nodes
-		nodes := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
-		gomega.Expect(len(nodes.Items)).NotTo(gomega.BeZero(), "No available nodes for scheduling")
-
-		// Cap max number of nodes
-		maxLen := len(nodes.Items)
-		if maxLen > maxNodes {
-			maxLen = maxNodes
-		}
```

Comment:
Is it fine to remove this capping way?

Reply:
The new code uses the new `GetBoundedReadySchedulableNodes`, which does the capping for you